src/main.js

```diff
@@ -2,18 +2,107 @@
+  descriptor2.innerHTML = randomCover.tagline2;
+});
 
 
 // Create your event handlers and other functions here 👇
 
+function getRandomIndex(covers) {
+  return Math.floor(Math.random() * covers.length);
+}
 
-// We've provided one function to get you started
-function getRandomIndex(array) {
-  return Math.floor(Math.random() * array.length);
+function createRandomCover(){
+  var randomImage = covers[getRandomIndex(covers)];
+  var randomTitle = titles[getRandomIndex(titles)];
+  var randomDescriptor1 = descriptors[getRandomIndex(descriptors)];
+  var randomDescriptor2 = descriptors[getRandomIndex(descriptors)];
+  var randomCover = new Cover(randomImage, randomTitle, randomDescriptor1, randomDescriptor2)
+  return randomCover
 }
+
+
+//Change visibility of the form
+document.querySelector('.make-new-button').addEventListener('click',function(){
+  document.getElementsByClassName('random-cover-button')[0].style.display = 'none';
+  document.getElementsByClassName('save-cover-button')[0].style.display = 'none';
+  document.getElementsByClassName('home-button')[0].style.display = 'block';
+  document.getElementsByClassName('view home-view')[0].style.display = 'none';
+  document.getElementsByClassName('view form-view hidden')[0].style.display = 'block';
+})
+
+//Saved covers button, iteration 2 
+document.querySelector('.view-saved-button').addEventListener('click',function(){
+  document.getElementsByClassName('random-cover-button')[0].style.display = 'none';
+  document.getElementsByClassName('save-cover-button')[0].style.display = 'none';
+  document.getElementsByClassName('home-button')[0].style.display = 'block';
+  document.getElementsByClassName('view home-view')[0].style.display = 'none';
+  document.getElementsByClassName('view form-view hidden')[0].style.display = 'none';
+  document.getElementsByClassName('saved-covers-section')[0].style.display = 'block';
+})
+//Home button, iteration 2 
+document.querySelector('.home-button').addEventListener('click',function(){
+  document.getElementsByClassName('random-cover-button')[0].style.display = 'block';
+  document.getElementsByClassName('save-cover-button')[0].style.display = 'block';
+  document.getElementsByClassName('home-button')[0].style.display = 'none';
+  document.getElementsByClassName('view home-view')[0].style.display = 'block';
+  document.getElementsByClassName('view form-view hidden')[0].style.display = 'none';
+  document.getElementsByClassName('saved-covers-section')[0].style.display = 'none';
+})
+
+//Create a new cover  
+
+// 1. Save the submitted data into the respective arrays (cover URL into the covers array, title string into the titles array, etc) so that future random covers can use the user-created data (Y)
+// 2. Use the values from the inputs to create a new instance of the Cover class (Y)
+// 3. Change back to the main home view (hiding the form view again) (Y)
+// 4. Display the newly created cover image, title, and descriptors in the main cover
```

Comment:
Great comments here! Stuff like this can be really helpful for a new dev jumping into the code base.